.pre-commit-hooks.yaml

```diff
@@ -65,6 +65,12 @@
     entry: '^\s*\.\. [a-z]+:$'
     language: pygrep
     types: [rst]
+-   id: rst-directive-space
+    name: rst directives space before two colons
+    description: 'Detect mistake of additional space in rst directive'
+    entry: '^\s*\.\. [a-z]+ ::$'
+    language: pygrep
+    types: [rst]
```

Comment:
this can probably get bundled in with the other rst check?

Reply:
OK! Done!